.github/workflows/build_test.yml

```diff
@@ -0,0 +1,62 @@
+name: Build/Test for PR and collaborator push
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '.github/workflows/build_test_publish.yml'
+      - '.github/workflows/changelog_test.yml'
+      - 'docker/**'
+      - 'doc/**'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '.github/workflows/build_test_publish.yml'
+      - '.github/workflows/changelog_test.yml'
+      - 'docker/**'
+      - 'doc/**'
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ubuntu_versions : [
+          22.04,
+          ]
+        pkg_mgr : [
+            apt,
+            conda
+        ]
+    
+    container:
+      image: ghcr.io/cyclus/cyclus_${{ matrix.ubuntu_versions }}_${{ matrix.pkg_mgr }}/${{ matrix.pkg_mgr }}-deps
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Building Cyclus
+        run: python install.py -j 2 --build-type=Release --core-version 999999.999999 
+
+      - name: Add bins to PATH
+        run: |
+          echo "PATH=${HOME}/.local/bin:$PATH" >> "$GITHUB_ENV"
+
+      - name: Add libs to LD_LIBRARY_PATH
+        run: |
+          echo "LD_LIBRARY_PATH=${HOME}/.local/lib:${HOME}/.local/lib/cyclus:$LD_LIBRARY_PATH" >> "$GITHUB_ENV"
+
+      - name: Cyclus Unit Tests
+        run: |
+          cyclus_unit_tests
+
+      - name: Cyclus Python Tests
+        run: |
+          cd tests && python -m pytest --ignore test_main.py
```

Comment:
Do you think it's important that these are different tasks in the job, or could they be just multiple lines on the same task

Reply:
I think it's important that the tests are separate steps so it is clear where failures happen.  I will throw the PATH and LD_LIBRARY_PATH modification in with the "Building Cyclus" step, those being their own steps is a remnant of my debugging